src/SwedbankPay.Sdk/Payments/TrustlyPayments/TrustlyPaymentOperations.cs

```diff
@@ -0,0 +1,61 @@
+﻿using SwedbankPay.Sdk.Extensions;
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+
+namespace SwedbankPay.Sdk.Payments.TrustlyPayments
+{
+    public class TrustlyPaymentOperations : OperationsBase
+    {
+        public TrustlyPaymentOperations(OperationList operations, HttpClient client)
+        {
+            foreach (var httpOperation in operations)
+            {
+                switch (httpOperation.Rel.Value)
```

Comment:
Codacy found an issue: [Add a 'default' clause to this 'switch' statement.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5814474)